Documentation/ApiOverview/BackendModules/BackendModuleApi/Index.rst

```diff
@@ -42,6 +42,8 @@ using the following API:
             'access' => 'admin',
             'icon' => 'EXT:beuser/Resources/Public/Icons/module-beuser.svg',
             'labels' => 'LLL:EXT:beuser/Resources/Private/Language/locallang_mod.xlf',
+            'navigationComponentId' => 'TYPO3/CMS/Backend/PageTree/PageTreeElement',
+            'inheritNavigationComponentFromMainModule' => true
```

Comment:
Here we should use `false`, otherwise this doesn't make sense since a custom navigation component was configured.

Also please add a trailing comma. :-)

Reply:
Agreed.